releases/mainnet/1.9.2/index.html

```diff
@@ -0,0 +1,25 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.9.2</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.9.2</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.9.2/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmXXTKKP8odp768F4EShHaFGsMmv55Dz1gsxTesKkfYUgU">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.9.2/abi/index.html">ABIs</a></li>
+				<li>Links to GitHub repositories
+					<ul>
+						<li>DSS deploy scripts (<a href="https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.9.2" target="_blank">changelog</a>)</li>
```

Comment:
GitHub release not available under that link. Either comment out or create release description

Reply:
I'll wait for full signoff here so I can tag it.

Reply:
There won't be a 1.9.2 for the scripts, so this should be commented out.

Reply:
Oh, yes, I missed that this was on deploy scripts.  I'll comment out.

Reply:
You can look again, I pushed an update.